src/plugins/gui/fiatPluginTypes.ts

```diff
@@ -35,7 +35,7 @@ export interface FiatPluginEnterAmountParams {
   label1: string
   label2: string
   convertValue: (sourceFieldNum: number, value: string) => Promise<string | undefined>
-  onSubmit: (value: FiatPluginEnterAmountResponse) => Promise<void>
+  onSubmit: (value: FiatPluginEnterAmountResponse) => void
```

Comment:
We generally want things like `onSubmit` to accept a promise, and then show a spinner as long as the promise is pending. This provides feedback to the user with essentially zero developer effort. It also comes with the expectation that the scene or component will catch any errors and do `showError` on the callback's behalf. Otherwise you have to remember to do that manually, which humans are pretty bad at.